pages/token-unit-conversion.tsx

```diff
@@ -26,7 +29,7 @@ export default function TokenUnitConversion() {
       value = parseInt(value, 16).toString()
     }
 
-    tokenPrecision.base = parseInt(decimal === '' ? tokenPrecision.base.toString() : '6')
+    tokenPrecision.base = parseInt(decimal || '18')
```

Comment:
no magic numbers. Please extract `18` to constant (ex. `DEFAULT_PRECISION`) and move it to the top of the file.